examples/example_quadruped_mp/main.cpp

```diff
@@ -304,7 +304,7 @@ int main(int argc, char **argv) {
       Pose3(Rot3(), Point3(0.14, 0, 0)); // Foot is 14cm along X in COM
   Pose3 bTw_i = wTb_i.inverse();
   for (auto &&leg : swing_sequence) {
-    const Pose3 bTfoot = bTw_i * robot.link(leg)->wTcom() * comTfoot;
+    const Pose3 bTfoot = bTw_i * robot.link(leg)->bTcom() * comTfoot;
```

Comment:
This is highly suspicious now: w(orld) index does not line up with b(ody) index. Investigate !

Reply:
wTb was set to be the bTcom transformation of the "body" link of the robot. Its the identity in this robot because the body link COM is the base frame. I guess there could be a robot with a "body" link with a COM frame which is different than the base frame (I havn't seen one), so I am not sure that its always identity. In this case , I think the bTw_i could be removed from this equation (I also checked to ensure that wTb_i is identity).

Reply:
Yes, I think this should just be removed, probably? But before we merge - and this is synergistic with your goals - you might want to fully understand what happens in this main.cpp file, and establish that it runs correctly both in master *and* in your branch.

Reply:
removed the bTw_i in this equation but its still there in the main.cpp in may places. I will go over all of these.